src/main/java/com/brave/playlist/PlaylistVideoService.kt

```diff
@@ -373,9 +380,10 @@ class PlaylistVideoService : Service(), Player.Listener, SessionAvailabilityList
 
     private fun sendCastStatusBroadcast(shouldShowControls: Boolean) {
         val intent = Intent()
-        intent.action = CAST_ACTION
-        intent.putExtra(SHOULD_SHOW_CONTROLS, shouldShowControls)
-        sendBroadcast(intent)
+        intent.action = packageName+CAST_ACTION
+        intent.putExtra(packageName+ SHOULD_SHOW_CONTROLS, shouldShowControls)
```

Comment:
`Extra` field names like `SHOULD_SHOW_CONTROLS` don't need to be changed this way, only intent actions. But you can leave it as it is.

Reply:
We need to use it as toggle for cast operations. but anyways i am removing it with viewmodel changes in upcoming commit